src/screens/game_map.py

```diff
@@ -340,87 +343,76 @@ def _setup_collision_rect(
         if SETUP_PATHFINDING:
             self._add_pf_matrix_collision((obj.x, obj.y), (obj.width, obj.height))
 
-    def _setup_collideable_object(
-        self,
-        pos: tuple[int, int],
-        obj: TiledObject,
-        layer: Layer,
-        groups: tuple[pygame.sprite.Group, ...] | pygame.sprite.Group,
+    def _setup_tree(
+        self, pos: tuple[int, int], obj: TiledObject, object_type: MapObjectType
     ):
-        """
-        Create a new collideable Sprite from the given TiledObject.
-        If this map's MapObjects instance doesn't contain this object's GID,
-        this method will raise an exception
-        """
-        object_type = self._map_objects[obj.gid]
-
-        CollideableMapObject(pos, object_type, z=layer).add(groups)
-
-        if SETUP_PATHFINDING:
-            self._add_pf_matrix_collision(
-                (
-                    obj.x + object_type.hitbox.x / SCALE_FACTOR,
-                    obj.y + object_type.hitbox.y / SCALE_FACTOR,
-                ),
-                (
-                    object_type.hitbox.width / SCALE_FACTOR,
-                    object_type.hitbox.height / SCALE_FACTOR,
-                ),
+        props = obj.properties
+        if props.get("size") == "medium" and props.get("breakable"):
```

Comment:
I feel it would be conventionally better to check if `props.get("breakable") is not None`, if you were relying on the fact `dict.get` returns None if the key does not exist

Reply:
You're right, but it doesn't only check if the property exist but also whether it is set to True or not